src/theme/space.stories.mdx

```diff
@@ -0,0 +1,41 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import { Figma } from "storybook-addon-designs/blocks"
+
+import { SpaceScale } from '../../.storybook/blocks/SpaceScale';
+import space from '../theme/space'
+
+<Meta title="Theme/Space" />
+
+# Space
+
+Spacing is an important part of design system. Instead of using arbitrary amounts of spacing
+in our components and layouts, we are limiting ourselves to standardised spacing scale.
+
+## Scale
+
+Our scale is based on "power of two" rule with several custom steps to smooth the scale. Each of
+the customized steps is marked with the `Plus` suffix.
+
+<SpaceScale scale={space} />
+
+## Stacking
+
+For stacking we can use any value from our scale and we can stack both vertically and horizontally.
+In code the stacking is implemented by margins.
+
+<Figma url="https://www.figma.com/file/vnzYfrwIISRYkr2iYE5TcD/Platform-Atoms?node-id=2314%3A51" />
+<Figma url="https://www.figma.com/file/vnzYfrwIISRYkr2iYE5TcD/Platform-Atoms?node-id=2314%3A53" />
+
+## Inseting
+
+We have three different kinds of inset:
+  1. `square` - vertical and horizontal inset are equal
+  2. `squish` - vertical inset is half of horizontal
+  3. `stretch` - horizontal inset is half of vertical
```

Comment:
Not sure I like this opinionated approach instead of being able to just set `paddingX="xs" paddingY="xxs"`.

But I guess being strict works for us, especially since we have several independent teams

Reply:
It should be ok as these three concepts will cover the majority of use-cases. If you have some legit use-case for some special combination of padding you are always able to do it in CSS and use whatever value from scale with `getSpace`.
Recommend https://medium.com/eightshapes-llc/space-in-design-systems-188bcbae0d62 where we took inspiration about the spacing scale and those three options.

Once the layout primitives, we should be able to cover most of the components with composition.